core/unit_test/TestGraph.hpp

```diff
@@ -564,10 +564,16 @@ TEST_F(TEST_CATEGORY_FIXTURE(graph), end_of_submit_control_flow) {
   //      their policy if the defaulted graph implementation is used.
   graph.submit(exec_3);
 
+  // clang-format off
   Kokkos::parallel_for(
       policy_t(exec_3, 0, 1),
+#if defined(KOKKOS_COMPILER_GNU) && (1010 == KOKKOS_COMPILER_GNU)
```

Comment:
How sure are we it's only 10.1? (i.e. neither 10.0 nor 10.2 have the issue)?

Reply:
Yes, it was only for `GCC 10.1` from my experiments.

Reply:
We check in the compiler explorer.  Maybe Romin can share a link.